RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -582,3 +582,133 @@ void auto_autumnatonQuest()
 
 	return;
 }
+
+boolean auto_haveTrainSet()
+{
+	return auto_get_campground() contains $item[model train set] && auto_is_valid($item[model train set]); //check if the model train set is in the campground
+}
+
+void auto_modifyTrainSet(int one, int two, int three, int four, int five, int six, int seven, int eight)
+{
+	string page = `choice.php?pwd&whichchoice=1485&option=1&slot[0]={one}&slot[1]={two}&slot[2]={three}&slot[3]={four}&slot[4]={five}&slot[5]={six}&slot[6]={seven}&slot[7]={eight}`;
```

Comment:
Do we have to have a check to ensure we can change the train set? Believe it is locked for 40 advs after changing it or something

Reply:
Yes. This gets called only from auto_checkTrainSet if the page has a button that says "Save Train Set Configuration". I did need to update autoscend_header to make sure it had the proper fuction name.